test/puppetlabs/http/client/sync_plaintext_test.clj

```diff
@@ -236,6 +258,32 @@
             (is (= "Hello, World!" (slurp (:body response))))))
         (.close client)))))
 
+(deftest java-api-cookie-test
+  (testlogging/with-test-logging
+    (testutils/with-app-with-config app
+      [jetty9/jetty9-service test-cookie-service]
+      {:webserver {:port 10000}}
+      (let [client (Sync/createClient (ClientOptions.))]
+        (testing "Set a cookie using Java API"
+          (let [response (.get client (RequestOptions. "http://localhost:10000/cookietest"))]
```

Comment:
@cprice404 refactored into this. i kept the  `let [client (Sync/createClient (ClientOptions.))]` around since putting it in the .get would make the line a little long, unless you think it would be better that way